client/signup/config/flows-pure.js

```diff
@@ -18,6 +18,14 @@ export function generateFlows( {
 	getDIFMSiteContentCollectionDestination = noop,
 } = {} ) {
 	const flows = [
+		{
+			name: HOSTING_LP_FLOW,
+			steps: [ 'plans', 'user', 'domains' ],
+			destination: getSignupDestination,
+			description: 'Create an account and a blog and then add the starter plan to the users cart.',
```

Comment:
These don't really get used anywhere but:

```suggestion
			description: 'Create an account and a blog and give the user the option of adding a domain and plan to the cart.',
```